templates/conf.d/mime.types.erb

```diff
@@ -0,0 +1,6 @@
+# MANAGED BY PUPPET
```

Comment:
Please convert this to an [epp](https://puppet.com/docs/puppet/5.5/lang_template_epp.html) template.